integration-tests/browser_integration_test.go

```diff
@@ -0,0 +1,167 @@
+// Package integrationtests contains broad integration integrationtests that
+// include a bootstrap node, a standalone node, and a browser node.
+package integrationtests
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/0xProject/0x-mesh/rpc"
+	"github.com/0xProject/0x-mesh/scenario"
+	"github.com/0xProject/0x-mesh/zeroex"
+	"github.com/chromedp/chromedp"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBrowserIntegration(t *testing.T) {
+	if !browserIntegrationTestsEnabled {
+		t.Skip("Integration tests are disabled. You can enable them with the --integration flag")
+	}
+
+	teardownSubTest := setupSubTest(t)
+	defer teardownSubTest(t)
+
+	// Declare a context that will be used for all child processes, servers, and
+	// other goroutines.
+	ctx, cancel := context.WithTimeout(context.Background(), 180*time.Second)
+	ctx, _ = chromedp.NewContext(ctx, chromedp.WithErrorf(t.Errorf))
+	defer cancel()
+
+	removeOldFiles(t, ctx)
+	buildForTests(t, ctx)
+
+	// wg is a WaitGroup for the entire tests. We won't exit until wg is done.
+	wg := &sync.WaitGroup{}
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		startBootstrapNode(t, ctx)
+	}()
+
+	// standaloneLogMessages is a channel through which log messages from the
+	// standalone node will be sent. We use a large buffer so it doesn't cause
+	// goroutines to block.
+	standaloneLogMessages := make(chan string, 1024)
+	// count := int(atomic.AddInt32(&nodeCount, 1))
```

Comment:
Is this commented out code temporary?

Reply:
Yes, I'm currently fixing some bugs in this PR.